site/en/docs/webstore/about_webstore/index.md

```diff
@@ -21,11 +21,13 @@ build Chrome-specific features into it.
 
 ## How do I start?
 
-1.  Learn more about [Chrome Extensions][4]
-2.  Publish your first extension by following the steps in [Publish in the Chrome Web Store][5]
+1.  Learn more about [Chrome Extensions][extensions]
+1.  Publish your first extension by following the steps in [Publish in the Chrome Web Store][cws-publish].
+1.  For addition help, check the [Chrome Web Store FAQs][cws-faq]
 
-[1]: http://chrome.google.com/webstore
-[3]: https://chrome.google.com/webstore/developer/dashboard
-[4]: /docs/extensions/mv3/overview/
-[5]: /docs/webstore/publish
+[cws-faq]: /docs/webstore/faq/
+[cws-publish]: /docs/webstore/publish
+[dashboard]: https://chrome.google.com/webstore/developer/dashboard
+[extensions]: /docs/extensions/mv3/overview/
+[webstore]: http://chrome.google.com/webstore
```

Comment:
update to https:// (we should probably automate this, but you're here changing stuff)